Tem-Test/server.R

```diff
@@ -5,22 +5,21 @@ library(ggplot2)
 server<- function(input,output) {
   dataset<- reactive({
     selected_data<- data_list[as.integer(input$dataset),]
-    browser()
-    get(selected_data["Item"],pasteO("package:", selected_data["Package"]))
+    get(selected_data["Item"],paste0("package:", selected_data["Package"]))
```

Comment:
Here I removed the no longer needed `browser()` call. You can put it back in if you need it later, of course.